bucketeer/src/androidTest/kotlin/io/bucketeer/sdk/android/e2e/BKTClientVariationTest.kt

```diff
@@ -72,9 +72,10 @@ class BKTClientVariationTest {
       BKTEvaluation(
         id = "feature-android-e2e-string:3:bucketeer-android-user-id-1",
         featureId = FEATURE_ID_STRING,
-        featureVersion = 3,
+        featureVersion = 4,
```

Comment:
Because I added the variation name on the console UI, it increased the feature version.
Same for the other tests.